src/main/java/org/htmlunit/html/HtmlFormUtil.java

```diff
@@ -98,13 +98,18 @@ public static List<HtmlElement> getSubmitButtons(final HtmlForm htmlForm) throws
     }
 
     /**
-     * Gets the first {@code <input type="submit">} element in this form.
+     * Gets the first {@code <input type="submit">} or {@code <button name="Submit">} element in this form.
      */
     public static HtmlElement getSubmitButton(final HtmlForm htmlForm) throws ElementNotFoundException {
         List<HtmlElement> submitButtons = getSubmitButtons(htmlForm);
         if (!submitButtons.isEmpty()) {
             return submitButtons.get(0);
         }
+        for (HtmlElement element : htmlForm.getElementsByAttribute("button", "name", "Submit")) {
+            if(element instanceof HtmlButton) {
```

Comment:
```suggestion
            if (element instanceof HtmlButton) {
```

Yeah I know https://github.com/jenkinsci/jenkins-test-harness/blob/07a24aeb0e8ba913a776d34700f721e357581590/src/main/java/org/htmlunit/html/HtmlFormUtil.java#L114 maybe we should enable Spotless.